README.md

```diff
@@ -7,13 +7,13 @@
 
 Docker Builder Image for cross-building Golang Prometheus projects.
 
-- `latest`, `main`, `1.5-main`, `1.5.4-main` ([1.5/main/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.5/main/Dockerfile))
-- `arm`, `1.5-arm`, `1.5.4-arm` ([1.5/arm/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.5/arm/Dockerfile))
-- `powerpc`, `1.5-powerpc`, `1.5.4-powerpc` ([1.5/powerpc/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.5/powerpc/Dockerfile))
-- `1.6-main`, `1.6.2-main` ([1.6/main/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.6/main/Dockerfile))
-- `1.6-arm`, `1.6.2-arm` ([1.6/arm/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.6/arm/Dockerfile))
-- `1.6-powerpc`, `1.6.2-powerpc` ([1.6/powerpc/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.6/powerpc/Dockerfile))
-- `1.6-mips`, `1.6.2-mips` ([1.6/mips/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.6/mips/Dockerfile))
+- `1.5-main`, `1.5.4-main` ([1.5/main/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.5/main/Dockerfile))
+- `1.5-arm`, `1.5.4-arm` ([1.5/arm/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.5/arm/Dockerfile))
+- `1.5-powerpc`, `1.5.4-powerpc` ([1.5/powerpc/Dockerfile](https://github.com/prometheus/golang-builder/blob/master/1.5/powerpc/Dockerfile))
```

Comment:
No objections, but I wonder why? It's not about keeping the images available in the docker hub, but about starting the list with deprecated images. People will be able to continue using them even if we remove them from the readme, right? If you want to keep them on the mage, maybe list them below the 1.6 images?